docs/docs/sinks/httpv2-sink.md

```diff
@@ -0,0 +1,7 @@
+# HttpV2 Sink
+
+HttpV2 Sink is implemented in Firehose using the Http sink connector implementation in Depot library. For details on all the features supported by HttpV2 Sink, please refer the Depot documentation [here](https://github.com/goto/depot/blob/main/docs/sinks/http.md).
+
+### Configuration
+
+For HttpV2 sink in Firehose we need to set first (`SINK_TYPE`=`httpv2`). There are some generic configs which are common across different sink types which need to be set which are mentioned in [generic.md](../advance/generic.md). Http sink specific configs are mentioned in Depot repository. You can check out the Http Sink configs [here](https://github.com/goto/depot/blob/main/docs/reference/configuration/http.md)
```

Comment:
can you update this doc link also - https://github.com/goto/depot/blob/main/docs/reference/configuration/http.md . it is non existant.

Reply:
@bsushmith that link is a placeholder for the Depot documentation. We'll start working on the Depot documentation next week, then that link will start working